src/Core/Annotation/ApiResource.php

```diff
@@ -218,6 +222,8 @@ public function __construct(
         ?string $securityMessage = null,
         ?string $securityPostDenormalize = null,
         ?string $securityPostDenormalizeMessage = null,
+        ?string $securityPostValidation = null,
+        ?string $securityPostValidationMessage = null,
```

Comment:
please add this to `src/Metadata/ApiResource`

Reply:
✅